tests/js/common/aql/aql-view-arangosearch-ddl-cluster.js

```diff
@@ -69,14 +77,48 @@ function IResearchFeatureDDLTestSuite() {
       }
     },
 
+    testViewIsBuilding: function () {
+      if (isServer) {
```

Comment:
Suggest to add `internal.debugClearFailAt();` to make test stable for failures in other